src/templates/rust-all/.gitignore.tftpl

```diff
@@ -5,10 +5,20 @@
 # will have compiled files and executables
 debug/
 target/
+
+*.env
+
 # Local cargo cache generated by make targets
 .cargo/registry
 
-# Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
+%{ if type == "svc" }
+server/.cargo/registry
+client/.cargo/registry
```

Comment:
Do we want to have separate .cargo dirs for server and client or can they be shared in the parent dir?
In the benchmarks repo we have the .cargo dir sitting in the parent dir and none in the package dirs

Reply:
Good point will combine to parent dir

Reply:
https://github.com/Arrow-air/tf-github/pull/34/commits/8f710a84089f9d4cd7907b447d47d911e0caf123